src/Uno.UI.RemoteControl.Server.Processors/HotReload/MetadataUpdates/CompilationWorkspaceProvider.cs

```diff
@@ -130,6 +141,45 @@ public static void InitializeRoslyn(string? workDir)
 			}
 		}
 
+		[GeneratedRegex(@"^net(\d+)(\.0)?$", RegexOptions.IgnoreCase | RegexOptions.Compiled, "en-CA")]
+		private static partial Regex NetVersionRegex();
+
+		[GeneratedRegex(@"^net(\d+)", RegexOptions.IgnoreCase | RegexOptions.Compiled, "en-CA")]
+		private static partial Regex NetMajorRegex();
```

Comment:
@carldebilly this one is never used at the moment, and there is a current CI build error for it

`/__w/1/s/src/Uno.UI.RemoteControl.Server.Processors/HotReload/MetadataUpdates/CompilationWorkspaceProvider.cs(148,32): error IDE0051: Private member 'CompilationWorkspaceProvider.NetMajorRegex' is unused (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0051) `